neurom/view/tests/test_view.py

```diff
@@ -30,11 +30,12 @@
 
 import itertools as it
 import numpy as np
+from numpy.testing import assert_array_almost_equal
```

Comment:
Can we use `np.testing.assert_array_almost_equal` instead of importing it separately? Otherwise should we import separately `assert_allclose`? Currently we use `np.testing.assert_allclose`